openmetadata-ui/src/main/resources/ui/src/pages/Application/ApplicationPage.tsx

```diff
@@ -0,0 +1,144 @@
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+import { Button, Col, Row } from 'antd';
+import { AxiosError } from 'axios';
+import { uniqueId } from 'lodash';
+import React, { useCallback, useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useHistory } from 'react-router-dom';
+import ApplicationCard from '../../components/Applications/ApplicationCard/ApplicationCard.component';
+import NextPrevious from '../../components/common/next-previous/NextPrevious';
+import { PagingHandlerParams } from '../../components/common/next-previous/NextPrevious.interface';
+import PageHeader from '../../components/header/PageHeader.component';
+import Loader from '../../components/Loader/Loader';
+import { ROUTES } from '../../constants/constants';
+import { PAGE_HEADERS } from '../../constants/PageHeaders.constant';
+import { App } from '../../generated/entity/applications/app';
+import { Paging } from '../../generated/type/paging';
+import { usePaging } from '../../hooks/paging/usePaging';
+import { getApplicationList } from '../../rest/applicationAPI';
+import { showPagination } from '../../utils/CommonUtils';
+import { getEntityName } from '../../utils/EntityUtils';
+import { getApplicationDetailsPath } from '../../utils/RouterUtils';
+import { showErrorToast } from '../../utils/ToastUtils';
+
+const ApplicationPage = () => {
+  const { t } = useTranslation();
+  const {
+    currentPage,
+    paging,
+    pageSize,
+    handlePagingChange,
+    handlePageChange,
+    handlePageSizeChange,
+  } = usePaging();
+  const history = useHistory();
+  const [isLoading, setIsLoading] = useState(true);
+  const [applicationData, setApplicationData] = useState<App[]>();
+
+  const fetchApplicationList = useCallback(async (pagingOffset?: Paging) => {
+    try {
+      setIsLoading(true);
+      const { data, paging } = await getApplicationList({
+        after: pagingOffset?.after,
+        before: pagingOffset?.before,
+        limit: pageSize,
+      });
+
+      setApplicationData(data);
+      handlePagingChange(paging);
+    } catch (err) {
+      showErrorToast(err as AxiosError);
+    } finally {
+      setIsLoading(false);
+    }
+  }, []);
+
+  const handleBotPageChange = ({
+    currentPage,
+    cursorType,
+  }: PagingHandlerParams) => {
+    handlePageChange(currentPage);
+    cursorType &&
+      fetchApplicationList({
+        [cursorType]: paging[cursorType],
+        total: paging.total,
+      } as Paging);
+  };
+
+  const viewAppDetails = (item: App) => {
+    history.push(getApplicationDetailsPath(item.fullyQualifiedName ?? ''));
+  };
+
+  const handleAddApplication = () => {
+    history.push(ROUTES.MARKETPLACE);
+  };
+
+  useEffect(() => {
+    fetchApplicationList();
+  }, []);
+
+  if (isLoading) {
+    return <Loader />;
+  }
```

Comment:
No data placeholder is missing, Not sure if required 
For Card layout we can use Card based skeleton instead Loader
Reder Persona listing page